analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -221,8 +221,12 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
 
         // Filter local repositories, as remote artifacts should never point to files on the local disk.
         val remoteRepositories = allRepositories.filterNot { it.url.startsWith("file:/") }
-        log.debug { "Found potential repositories for '$artifact': $remoteRepositories" }
-        log.debug { "Ignoring local repositories: ${allRepositories - remoteRepositories}" }
+        val localRepositories = allRepositories - remoteRepositories
+        if (localRepositories.isNotEmpty()) {
+            log.debug { "Ignoring local repositories $localRepositories." }
+        }
```

Comment:
You could wrap this with `if (log.isDebugEnabled)` to not compute `localRepositories` when it is not used.

Reply:
Done.